### DISCUSSION TITLE
Charts in Insights show incorrect count

### DISCUSSION THREAD

**Discussion by lalitindoria** [community/community#42422] [Projects and Issues]
### Select Topic Area

Bug

### Body

The filter criteria results in 37 issues but the chart is plotted without considering the filters. It makes Insights unusable.

<img width="860" alt="image" src="https://user-images.githubusercontent.com/7344511/209210973-0e36fc6d-e22e-44e5-b65e-10f97e378f32.png">

**Comment by chiedo** [comment]
Thanks for sharing this feedback @lalitindoria! I'll share it with the right folks.

**Reply by mehmet-kaya-1994** [reply]
Excluding status is not correctly working. I have currently about 10 Issues in Status "Fertig". 

Status "Fertig" are issues which are also closed.

Be aware green icon of "Fertig" is confusing with blue column in chart.

![image](https://user-images.githubusercontent.com/61547085/213151581-e5d73bb6-5dce-439f-ac34-cef8ea57e429.png)

**Reply by chiedo** [reply]
I haven't been able to reproduce this @mehmet-kaya-1994 but I'll keep an eye open to see if I can catch this bug in the wild!

**Reply by aboterolh** [reply]
Just to chime in that we still see the behavior described by the original issue here. Are there examples of other people’s Insights dashboards? Not sure how everyone else is doing it, because this proper filtering I think it’s required by most filters. 🤔 

Thanks!
